### PR TITLE
Removing 'master' in platform get command

### PR DESCRIPTION
**Pull request by matthiaz**
Changing 'master' to project->default_branch so we don't mention master if they specified something else.

Also, attempt to move to the correct branch after the init.

### CODE REVIEW

File: src/Command/Project/ProjectGetCommand.php [hunk 1]
```diff
@@ -137,15 +137,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->debug('Initializing the project');
             $localProject->mapDirectory($projectRoot, $project);
 
+            if($git->getCurrentBranch($projectRoot) != $project->default_branch) {
+                $this->debug('current branch does not match the default_branch, create it.');
+                $git->checkOutNew($project->default_branch, null, null, $projectRoot);
+            }
```
Comment (pjcdawkins) [inline src/Command/Project/ProjectGetCommand.php:143]: We should probably try to pass `--initial-branch` to the `git init` call above, maybe only if Git is version 2.28 or above.

https://git-scm.com/docs/git-init/2.28.0

Comment (matthiaz) [reply src/Command/Project/ProjectGetCommand.php:143]: https://github.com/platformsh/platformsh-cli/pull/1077/files#diff-758a78d9bf121fd02f690e51541be9c098d2ad2ee91f1f991e885162b30a2b68R161 like this then?

File: src/Service/Git.php [hunk 2]
```diff
@@ -151,13 +151,17 @@ public function execute(array $args, $dir = null, $mustRun = false, $quiet = tru
      *
      * @return bool
      */
-    public function init($dir, $mustRun = false)
+    public function init($dir, $initial_branch='master', $mustRun = false)
```
Comment (pjcdawkins) [inline src/Service/Git.php:154]:
```suggestion
    public function init($dir, $initial_branch = '', $mustRun = false)
```

File: src/Service/Git.php [hunk 3]
```diff
@@ -151,13 +151,17 @@ public function execute(array $args, $dir = null, $mustRun = false, $quiet = tru
...
     {
         if (is_dir($dir . '/.git')) {
             throw new \InvalidArgumentException("Already a repository: $dir");
         }
 
-        return (bool) $this->execute(['init'], $dir, $mustRun, false);
+        $args = ['init'];
+        if ($this->supportsGitInitialBranchFlag()) {
```
Comment (pjcdawkins) [inline src/Service/Git.php:161]:
```suggestion
        if ($initial_branch !== '' && $this->supportsGitInitialBranchFlag()) {
```

File: src/Command/Project/ProjectGetCommand.php (hunk 1 above)
Comment (pjcdawkins) [reply src/Command/Project/ProjectGetCommand.php:143]: Exactly

**Comment by pjcdawkins** [comment]
thanks @matthiaz

**Pull request merged**